package.json

```diff
@@ -3,13 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.2.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "@types/jest": "^24.0.0",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "antd": "^4.6.4",
+    "graphql": "^15.3.0",
+    "moment": "^2.28.0",
```

Comment:
https://momentjs.com/docs/#:~:text=We%20now%20generally%20consider%20Moment,adding%20new%20features%20or%20capabilities.

dayjs or date-fns